scripts/test-sim-dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+docker run --rm -p 5432:5432 --name storj_sim_postgres postgres &
```

Comment:
Wondering whether this should bind postgres to a non-standard postgres support since users might have a postgres instance for other things already setup.

Reply:
yeah I agree. Ideally, it should choose random free port but as far I remember selecting such port with bash is not a trivial task, or maybe I'm wrong?

Reply:
I was also thinking if the container name should be unique for each execution.

Reply:
You can use something like `5433` or similar... it'll be slightly better than the standard port. But yeah, making it select a random port will be really difficult.

Reply:
as long as you clean up the old instance there shouldn't be a problem. PS: I don't think you need to add `&` to the end of this line, because the container will run in the background anyways.

Reply:
to run in background I need to add `-d` but then something with waiting for start doesn't work for me. but i will take a look at this later.

Reply:
Ah nevermind, I think I got confused with some other thing.

Reply:
> to run in background I need to add `-d` but then something with waiting for start doesn't work for me. but i will take a look at this later.

I don't understand that if you run in `&` you don't need something with waiting for start while with `-d` you need it.

Reply:
turns out to be a bug in my code :) I will use `-d`

Reply:
done